### PR TITLE
Support for Gyroscope Limited Axes sensor

### PR DESCRIPTION
**Pull request by Chandra-Mauli-Sharma**
In ref. to #40.
- The support for the Gyroscope Limited Axes sensor is added.
- WIP tag added to Readme.md for Gyroscope Limited Axes sensor until approval.

### CODE REVIEW

File: composesensors/src/main/java/com/mutualmobile/composesensors/GyroscopeLimitedAxesSensorState.kt [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+class GyroscopeLimitedAxesSensorState internal constructor(
+    val xAxisSupported: Float = 0f,
+    val yAxisSupported: Float = 0f,
+    val zAxisSupported: Float = 0f,
+    val xRotation: Float = 0f,
+    val yRotation: Float = 0f,
+    val zRotation: Float = 0f,
+    val isAvailable: Boolean = false,
+    val accuracy: Int = 0,
+) {
```
Comment (shubhamsinghmutualmobile) [inline composesensors/src/main/java/com/mutualmobile/composesensors/GyroscopeLimitedAxesSensorState.kt:15]: Let's please name the file according to https://github.com/mutualmobile/ComposeSensors/issues/40 i.e. `LimitedAxesGyroscopeSensorState`, give it proper documentation, and use the same parameter names as the already implemented `GyroscopeSensorState` class for better API cohesion (please refer [GyroscopeSensorState.kt](https://github.com/mutualmobile/ComposeSensors/blob/main/composesensors/src/main/java/com/mutualmobile/composesensors/GyroscopeSensorState.kt)).

**Review by shubhamsinghmutualmobile — CHANGES_REQUESTED** [review]

File: composesensors/src/main/java/com/mutualmobile/composesensors/GyroscopeLimitedAxesSensorState.kt [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+    }
+
+    override fun toString(): String {
+        return "GyroscopeLimitedAxesSensorState(xAxisSupported=$xAxisSupported, yAxisSupported=$yAxisSupported, zAxisSupported=$zAxisSupported, xRotation=$xRotation, yRotation=$yRotation, zRotation=$zRotation, isAvailable=$isAvailable, accuracy=$accuracy)"
+    }
+
+}
+
+@Composable
+fun rememberGyroscopeLimitedAxesSensorState(
```
[the remark this answers is not in the document]
Comment (Chandra-Mauli-Sharma) [reply composesensors/src/main/java/com/mutualmobile/composesensors/GyroscopeLimitedAxesSensorState.kt]: I have made the PR resolving the requested changes. Sorry for the delay

**Review by shubhamsinghmutualmobile — APPROVED** [review]

**Comment by shubhamsinghmutualmobile** [comment]
Let's fix the ktlint issues and we should be good to go for this PR @Chandra-Mauli-Sharma ✅

**Comment by shubhamsinghmutualmobile** [comment]
Screenshot 2023-04-20 at 6 24 26 PM

@Chandra-Mauli-Sharma could you also please `Update branch` to the latest code from `development`?

**Comment by Chandra-Mauli-Sharma** [comment]
Done

**Review by shubhamsinghmutualmobile — APPROVED** [review]

**Pull request merged**